docs/reference/components/sources/generator.cue

```diff
@@ -41,19 +41,32 @@ components: sources: generator: {
 	}
 
 	configuration: {
-		batch_interval: {
+		format: {
+			description: "The format of the randomly generated output."
+			required:    true
+			warnings: []
+			type: string: {
+				enum: {
+					"random":        "Lines are chosen at random from the list specified using `lines`."
```

Comment:
I'm not sure I'm going to be a fan of `random` neither, as it sounds like the contents could be random. Something like `shuffle` or `selection` is closer, but still awkward.

Reply:
Shuffle seems better to me